plugins/modules/packaging/os/pkgin.py

```diff
@@ -181,11 +187,15 @@ def query_package(module, name):
             elif raw_state == '=' or raw_state == '>':
                 return 'present'
             else:
+                # Package found but not installed
```

Comment:
Using here either `absent` or `not_installed` (as string or enum) the comment would become as well obsolete